relay-server/src/utils/profile.rs

```diff
@@ -287,6 +287,110 @@ pub fn parse_typescript_profile(item: &mut Item) -> Result<(), ProfileError> {
     Ok(())
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct RustProfile {
+    duration_ns: u128,
+    platform: String,
+    architecture: String,
+    // string representation of Uuid v4
+    trace_id: String,
```

Comment:
Any reason `trace_id`,`transaction_id` and `profile_id` are not of type `EventId`, like in `AndroidProfile`?

https://github.com/getsentry/relay/blob/d8f887a595a3ee7c78c16d4513ab86242c859f50/relay-server/src/utils/profile.rs#L82

Maybe we could even define keys used in multiple profile structs in a common helper struct: https://serde.rs/attr-flatten.html#factor-out-frequently-grouped-keys

Reply:
That's a good point. Thanks for bringing this up.
Changed all of them to `EventID` as I don't have specific reasons to keep them as String